pom.xml

```diff
@@ -59,8 +53,50 @@
     <maven.gpg.plugin.version>1.6</maven.gpg.plugin.version>
   </properties>
 
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <repository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+  </distributionManagement>
+
   <build>
     <plugins>
+      <plugin>
```

Comment:
The Sonatype POM also contains plugins which we need in order to meet the Sonatype requirements, these include:

* The source JAR plugin
* The GPG plugin
* The JavaDoc plugin which seem to have a different config in our OSS parent - we need the config as is in the Sonatype POM.

I don't see those plugins here, are they in our parent POM already? Perhaps move them to a profile like in the Sonatype POM